client/src/components/search.js

```diff
@@ -0,0 +1,71 @@
+import React,{Component} from 'react';
+import SearchBox from './searchbox';
+import '../stylesheets/css/index.css';
+import '../stylesheets/css/components/App.css'
+import InputSearchService from '../services/api';
+
+class SearchComponent extends Component{
+    constructor(props){
+        super(props)
+        this.state = {
+            placeholderText: 'Enter a member name..',
+            searchType: 'member',
+            inputValue: '',
+            searchResults: InputSearchService.fetchSearch('members')
+        }
+        this.setPlaceholder = this.setPlaceholder.bind(this);
+        this.updateInputValue = this.updateInputValue.bind(this);
+    }
+
+//FUNCTION TO BE FIRED ON SELECTOR CHANGE
+//CHECKS TARGET VALUE AND SETS PLACEHOLDER AND SEARCH TYPE STATE
+    setPlaceholder(event){
+        var target = event.target.value;
```

Comment:
I think you're already running es6, right? You're using a class definition for this component. If that is the case, you should not use `var` anymore. Use `let` or `const` for variable declaration.